src/odemis/acq/path.py

```diff
@@ -552,7 +549,13 @@ def _doSetPath(self, path, detector):
                                     self._stored[comp_role, axis] = comp.position.value[axis]
                                 break
                         else:
-                            logging.debug("Choice %s is not present in %s axis", pos, axis)
+                            if mode == "mirror-align" and pos == "pass-through":
+                                # That's typically a sign that the microscope file is incorrect
+                                # eg, a typo in the filter name, as if there is a filter wheel
+                                # and a ccd, there should always be a pass-through position
+                                logging.warning("No 'pass-through' filter available, alignment might be harder due to limited signal")
+                            else:
+                                logging.debug("Choice %s is not present in %s axis", pos, axis)
```

Comment:
What will happen when this choice is not available. Stick with the old one? Mention this in the debug message?

Reply:
If the choice is not present, is that an error in the yaml file? Should we hint that in the message?